.infrastructure/helm-chart/todoapp/templates/nodeport.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Chart.Name }}-nodeport
+  namespace: {{ .Values.todoapp.namespace }}
```

Comment:
The `todoapp.namespace` value is referenced here, but it is not defined in the `values.yaml` file. Ensure that this value is added to avoid deployment issues.